src/main.c

```diff
@@ -288,6 +291,46 @@ static uint32_t read_encoder_right(void)
 
 
 /**
+ * @brief Setup for adc: Four injected channels on scan mode for ADC1.
+ * - Initialize channel_sequence structure to map physical channels vs
+ * sw injected channels.
+ * - Power off the ADC to be sure that doesn't run during configuration.
+ * - Enable scan mode with single conversion mode triggered by software.
+ * - Configure the alignment (right) and the sample time (28.5 cycles of ADC
+ * clock)
+ * - Set injected sequence with channel_sequence structure
+ * - Power on the ADC and wait for ADC starting up (about 3 ms, See adc_power_on
+ * function from libopencm3)
+ * - Calibrate the ADC
```

Comment:
Not very important, but to be consistent with docstring styles:

- Maybe we could avoid contractions? (i.e.: "doesn't", "vs", "sw"...)
- Maybe try to always end the list items with a dot?
- Maybe add 2 spaces when continuing a list item in a second line? (i.e.: like in `setup_pwm` docstring)

Do not know if there is a script to check that as well to put it in Travis... :joy:

Reply:
Yes, I will try to include it on my editor